src/app/modules/user/user.controller.ts

```diff
@@ -206,3 +214,51 @@ export const handleFetchUser: ControllerHandler = async (req, res) => {
       return res.status(statusCode).json({ message: errorMessage })
     })
 }
+
+export const _handleUpdateUserLastSeenFeatureUpdateDate: ControllerHandler<
+  unknown,
+  string,
+  UpdateUserLastFeatureUpdateDateDto
+> = async (req, res) => {
+  const { latestLastSeenFeatureUpdateDate } = req.body
```

Comment:
nit: seems unnecessarily verbose; suggest just `date`, since the endpoint is already indicative of what is being updated